core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractSample.java

```diff
@@ -751,9 +751,10 @@ public Sample build() {
           return SampleImpl.sampleAnalyte(this);
         } else if (sampleTissue != null) {
           return SampleImpl.sampleTissue(this);
+        } else {
+          return SampleImpl.sampleTissueProcessing(this);
         }
       }
-      throw new IllegalArgumentException("No sample can be built with the specified parameters.");
```

Comment:
I did it this way because our TissueProcessing samples contain zero extra information (nor db tables) and we don't have a SampleTissueProcessing interface/implementation. Is creating those going to be the best solution, in your opinion?